### PR TITLE
Update to Velocity 1.0.0

### PR DESCRIPTION
**Pull request by js6pak**
This commit updating kyori text to 3.0 and **changing acf-velocity to its [fork](https://github.com/aikar/commands/pull/216)**
If you want you can wait with merging this until acf-velocity fix will be merged

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -166,10 +170,11 @@
...
             <version>0.5.0-SNAPSHOT</version>
         </dependency>
+        <!-- temporary fix for https://github.com/aikar/commands/pull/216 -->
         <dependency>
-            <groupId>co.aikar</groupId>
-            <artifactId>acf-velocity</artifactId>
-            <version>0.5.0-SNAPSHOT</version>
+            <groupId>com.github.egg82</groupId>
+            <artifactId>commands</artifactId>
+            <version>fix~velocity-SNAPSHOT</version>
```
Comment (xknat) [inline pom.xml]: ???

Comment (js6pak) [reply pom.xml]: as i said its fork from jitpack, but it just was merged so i will update this pr tomorrow

**Comment by McLive** [comment]
Thank you, I'll merge when ACF-velocity is updated! :)

**Comment by js6pak** [comment]
acf-velocity is finally deployed :)

**Pull request merged**